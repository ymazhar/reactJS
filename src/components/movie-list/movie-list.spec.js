import React from "react";
import MovieList from "./movie-list";


describe("Movie list component", () => {
    it("should render Movie list component", () => {
        const component = shallow(<MovieList/>);
        expect(component).toMatchSnapshot();
    })
})