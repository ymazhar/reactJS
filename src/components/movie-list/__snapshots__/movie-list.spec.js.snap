// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie list component should render Movie list component 1`] = `
<div
  className="movie-list"
/>
`;
